src/pybind/MatrixReader.py

```diff
@@ -0,0 +1,39 @@
+import kaldi_pybind as kp
+
+class SequentialVectorReader(kp.SequentialBaseFloatMatrixReader_Vector):
+    def __init__(self, rspecifier = None):
+        if not rspecifier:
+            kp.SequentialBaseFloatMatrixReader_Vector.__init__(self)      
+        else:
+            kp.SequentialBaseFloatMatrixReader_Vector.__init__(self, rspecifier)      
```

Comment:
It's great that you've done this!  However when I look at it now, I think this approach won't scale because there are are too many types of readers.  I was probably over-thinking the efficiency issue.  I'm thinking we could instead have the user use this syntax:
```
   for key,value in kaldi.ReaderIterator(kaldi.SequentialMatrixReader(rxfilename)):
        ...
```
  and kaldi could be a package with an __init__ file that contains:
```
    from kaldi_pybind import *
    from some_other_local_package import *   
    etc.
```
where that `some_other_local_package` would include the ReaderIterator class,
which would take the specific reader object as a function argument.

Reply:
Can I assume the specific reader objects are initialized with a given rspecifier?

> kaldi.ReaderIterator(kaldi.SequentialMatrixReader(rxfilename))